feat: add Windows installer and Microsoft Store distribution guides (#2589)
tauri-apps/tauri-docs: src/content/docs/distribute/microsoft-store.mdx

---
title: Microsoft Store
sidebar:
  order: 1
---

import CommandTabs from '@components/CommandTabs.astro';

Microsoft Store is the Windows app store operated by Microsoft.

This guide only covers details for distributing Windows Apps directly to the Microsoft Store.
See the [Windows Installer guide] for more information on Windows installer distribution options and configurations.

## Requirements

To publish apps on the Microsoft Store you must have a Microsoft account
and [enroll] as a developer either as an individual or as a company.

## Changing App Icon

The Tauri CLI can generate all icons your app needs, including Microsoft Store icons.
Use the `tauri icon` command to generate app icons from a single PNG or SVG source:

<CommandTabs
  npm="npm run tauri icon /path/to/app-icon.png"
  yarn="yarn tauri icon /path/to/app-icon.png"
  pnpm="pnpm tauri icon /path/to/app-icon.png"
  cargo="cargo tauri icon /path/to/app-icon.png"
/>

## Setting up

> After enrolling as a developer with your Microsoft account, the first step to distribute your Tauri app in the Microsoft Store
> is to register your app in the [Apps and Games] page.

Reviewer: ```suggestion
After you have enrolled as a developer with your Microsoft account you need to register your app in the [Apps and Games] page.
```
